x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -426,10 +426,14 @@ export const AddExceptionFlyoutWrapper: React.FC<AddExceptionFlyoutWrapperProps>
     return null;
   }, [maybeRule]);
 
+  const isESQLRule = enrichedAlert?.['kibana.alert.rule.parameters']?.type === 'esql';
```

Comment:
we need to cover ML rule here [too](https://github.com/elastic/kibana/issues/184377#issuecomment-2137151326)


https://github.com/elastic/kibana/assets/92328789/cc86867e-87de-4527-901f-d3941dfea49f